openstates/in/utils.py

```diff
@@ -0,0 +1,22 @@
+import requests
+
+
+def get_with_increasing_timeout(scraper,link,fail=False,kwargs={}):
```

Comment:
I believe it should; check out `scrapelib.RetrySession`, from which `scrapelib.Session` inherits:
https://github.com/sunlightlabs/scrapelib/blob/master/scrapelib/__init__.py#L171